src/otx/recipe/_base_/train.yaml

```diff
@@ -9,7 +9,7 @@ callbacks:
       check_on_train_epoch_end: false
       min_delta: 0.001
       warmup_iters: 30
-      warmup_epochs: 3
+      warmup_epochs: 7
```

Comment:
Yes in my opinion there shouldn't be any modifications to `/_base_/train.yaml`. If we need this, we need to update this in each model recipe.

Reply:
Okay, I will revert it, but to be honest, 7-10 epochs are reasonable value to start early stopping, 3 epochs are not enough for many models to stabilize

Reply:
reverted https://github.com/openvinotoolkit/training_extensions/pull/3951/commits/d3e910e76cf62f2f63f8ea5623041c6bafc4e9f9

Reply:
We cannot be sure until we test it experimentally because small epochs may be enough on small datasets because models only have to adapt to the distribution for a small number of samples.
But this is not in the scope of this PR that temporarily adds templates.